src/miner.cpp

```diff
@@ -120,6 +120,8 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn, CWallet* pwallet,
     else
         pblock->nVersion = 3;
 
+    pblock->nVersion = 5;   // Supports CLTV activation
```

Comment:
can we possibly clean up the preceding line's logic regarding the block version while we're at this?